Anchor.toml

```
@@ -2,28 +2,41 @@
 seeds = false
 skip-lint = false
 
-[programs.localnet]
-solana_ibc = "EnfDJsAK7BGgetnmKzBx86CsgC5kfSPcsktFCQ4YLC81"
-
 [programs.devnet]
-solana_ibc = "EnfDJsAK7BGgetnmKzBx86CsgC5kfSPcsktFCQ4YLC81"
+restaking = "8n3FHwYxFgQCQc2FNFkwDUf9mcqupxXcCvgfHbApMLv3"
+solana_ibc = "9fd7GDygnAmHhXDVWgzsfR6kSRvwkxVnsY8SaSpSH4SX"
+
+[programs.localnet]
+restaking = "8n3FHwYxFgQCQc2FNFkwDUf9mcqupxXcCvgfHbApMLv3"
+solana_ibc = "9fd7GDygnAmHhXDVWgzsfR6kSRvwkxVnsY8SaSpSH4SX"
 
 [registry]
 url = "https://api.apr.dev"
 
+[provider]
+cluster = "Localnet"
+wallet = "~/.config/solana/id.json"
+
 [workspace]
 members = [
+  "solana/restaking/programs/restaking",
   "solana/solana-ibc/programs/solana-ibc"
 ]
 
-[provider]
-cluster = "Localnet"
-wallet = "~/.config/solana/id.json"
-
 [scripts]
-# test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
-test = "cargo test --lib -- --nocapture --include-ignored ::anchor"
+# test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 solana/restaking/tests/**/*.ts"
+test = "sh test.sh"
```

Make `test.sh` executable and then:
```suggestion
test = "./test.sh"
```
`sh` is not correct here since `test.sh` is a Bash script. Alternatively, convert `test.sh` to be POSIX compliant.

---

makes sense.